docs/docs/forms.mdx

```diff
@@ -3,9 +3,7 @@ id: forms
 sidebar_label: Forms
 title: Forms
 description: Follow a rule-based process of information gathering using forms in open source bot framework Rasa.
-abstract: One of the most common conversation patterns is to collect a few pieces of
-          information from a user in order to do something (book a restaurant, call an
-          API, search a database, etc.). This is also called **slot filling**.
+abstract: One of the most common conversation patterns is to collect a few pieces of information from a user in order to do something (book a restaurant, call an API, search a database, etc.). This is also called **slot filling**.
```

Comment:
for the future: you don't need to put this all in one line (and shouldn't as this is hard to read in an editor that doesn't break lines like this one). the content above `---` is proper yaml, so you can use text blocks:
```yaml
abstract: |
  One of the most common conversation patterns is to collect a few 
  pieces of information from a user in order to do something (book a 
  restaurant, call an API, search a database, etc.). This is also called 
  **slot filling**.
```

Reply:
ah that is better thanks

Reply:
oooh this is good to know!